data/json/items/gun/shot.json

```
@@ -511,7 +511,7 @@
     "type": "GUN",
     "name": { "str": "Cobray Streetsweeper" },
     "description": "Less shotgun and more comically oversized revolver, the Cobray Streetsweeper sold poorly before it was deemed a destructive device.  The cylinder is driven by a clockspring, cannot be indexed by hand, and must be ejected with an ejector rod.  Its unique design allows for all 12 shells to be fired in under 3 seconds, as demonstrated by the ATF technical branch.",
-    "weight": "419 g",
+    "weight": "400 g",
```

Went with nearest 10 grams for this one instead since gun, also 420 teehee
```suggestion
    "weight": "420 g",
```